export/tests/disk/test_cli.py

```diff
@@ -454,3 +458,40 @@ def test_parse_correct_mountpoint_from_pexpect(self, mock_pexpect):
         )
         assert mv.unlocked_name == "/dev/dm-0"
         assert mv.mountpoint == "/media/usb"
+
+    def test__write_data_correct_permissions(self):
+        # This is the permission set when extracting the archive
+        old_umask = os.umask(0o77)
```

Comment:
I don't think CI and this test play well together, and maybe it's the kind of thing that should just be run locally..?

Reply:
CI runs the tests as root (in the container), which I suspect is screwing up the umask stuff. We can switch it to run as a regular user and set up passwordless sudo.

Reply:
Right. That's probably a good idea if it's not hugely inconvenient (and if messing around with umask on ci isn't a bad idea in the first place), because it would more closely emulate vm setup.

Reply:
Let me take a shot at enabling that...